src/components/PersonLink/PersonLink.tsx

```diff
@@ -0,0 +1,28 @@
+import { Link, useSearchParams } from 'react-router-dom';
+import cn from 'classnames';
+import { FC } from 'react';
+import { Person } from '../../types';
+
+interface Props {
```

Comment:
Similar names are okay for a small project, but on large projects we will have many interfaces and types, so it is better to give more detailed names, for example `PersonLinkProps`.
This applies to all code.